R/data-mlbbat10.R

```diff
@@ -30,58 +30,56 @@
 #' @source \url{https://www.mlb.com}, retrieved 2011-04-22.
 #' @keywords datasets
 #' @examples
-#' \dontrun{
+#' library(ggplot2)
+#' library(dplyr)
+#' library(scales)
 #' d   <- mlbbat10[mlbbat10$at_bat > 200,]
```

Comment:
Try recreating the figures without subsetting for `at_bat > 200`, do they look ok? If so, we can remove this and work with the full dataset. If, for some reason, they don't tell a good story we can keep this, but

- use `filter()` instead of the square brackets
- name the resulting data frame `mlbbat10_gt200` or something like that (more informative than `d`)